api/management-model/build.gradle.kts

```diff
@@ -50,13 +50,13 @@ val generatedOpenApiSrcDir = project.layout.buildDirectory.dir("generated-openap
 openApiGenerate {
   // The OpenAPI generator does NOT resolve relative paths correctly against the Gradle project
   // directory
-  inputSpec = specsDir.file("polaris-management-service.yml").asFile.absolutePath
+  inputSpec = provider { specsDir.file("polaris-management-service.yml").asFile.absolutePath }
   generatorName = "jaxrs-resteasy"
-  outputDir = generatedDir.get().asFile.absolutePath
+  outputDir = provider { generatedDir.get().asFile.absolutePath }
   modelPackage = "org.apache.polaris.core.admin.model"
-  ignoreFileOverride = rootDir.file(".openapi-generator-ignore").asFile.absolutePath
-  removeOperationIdPrefix = true
-  templateDir = templatesDir.asFile.absolutePath
+  ignoreFileOverride.set(provider { rootDir.file(".openapi-generator-ignore").asFile.absolutePath })
```

Comment:
This is because the property's declared as `Property<String?>` by the plugin extension. It doesn't make sense to declare nullable values as `Property`s - it has a value or not.